htop/Containerfile

```diff
@@ -0,0 +1,6 @@
+#Start from your base image. Here I'm using rhel 9.2 from the 4.14 image
```

Comment:
We should probably prefix every line here with a link to the docs, like
https://docs.openshift.com/container-platform/4.12/post_installation_configuration/coreos-layering.html#coreos-layering
which explains how to find the latest image.

(Hopefully in the future we'll have proper tags *outside* of OCP, but that's its own huge topic)

Reply:
Done - I also changed it 9.2 from a 4.13 nightly, which was a better fit